src/components/Card/Card.js

```diff
@@ -3,31 +3,35 @@ import { UserContext } from '../../context/UserContext';
 import './Card.scss';
 
 export const Card = ({ value, socket, disabled, selected }) => {
-  const { userObj } = useContext(UserContext);
+  const { userObj, setVote } = useContext(UserContext);
   const [isSelected, setIsSelected] = useState(false);
 
   useEffect(() => {
-	setIsSelected(selected);
-  }, [selected])
+    setIsSelected(selected);
+  }, [selected]);
 
   const emitValue = (value) => {
-	if(selected) {
-		socket.emit('vote', {...userObj, ...{vote: null}});
-	} else {
-		socket.emit('vote', {
-			...userObj,
-			...{
-				vote: value
-			}
-		});
-	}
-	setIsSelected(!isSelected);
+    if (selected) {
+      socket.emit('vote', { ...userObj, ...{ vote: null } });
+      setIsSelected(0);
+    } else {
+      socket.emit('vote', {
+        ...userObj,
+        ...{
+          vote: value,
+        },
+      });
+    }
+    setIsSelected(!isSelected);
+    setVote(value);
```

Comment:
Formatter ended up changing a lot but the only line I actually changed is adding in `setVote` so the vote value in UserContext would update so it could be read elsewhere 👍